rnd/autogpt_server/autogpt_server/blocks/llm.py

```diff
@@ -16,6 +17,7 @@
     "openai": BlockSecret("openai_api_key"),
     "anthropic": BlockSecret("anthropic_api_key"),
     "groq": BlockSecret("groq_api_key"),
+    "ollama": BlockSecret("ollama_api_key"),
```

Comment:
You can use BlockSecret(value="") instead and remove everything about ollama_api_keu

Reply:
oh sweet, thanks will do that